src/chrome/content/rules/Zillowstatic.com.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Zillowstatic.com">
+	<target host="zillowstatic.com" />
+	<target host="www.zillowstatic.com" />
+	<target host="photos1.zillowstatic.com" />
```

Comment:
Could you add a test url please? http://photos1.zillowstatic.com/i_g/ISatx8mbaxt0v7.jpg